test/jdk/sun/net/www/http/HttpClient/MultiThreadTest.java

```diff
@@ -280,14 +366,39 @@ public void run() {
      * multiple http requests on same connection.
      */
     class Worker extends Thread {
-        Socket s;
-        int id;
-        volatile boolean headerFound;
-
-        Worker(Socket s, int id) {
+        final long TIMEOUT = MultiThreadTest.KEEP_ALIVE; // seconds
+        final long KEEP_ALIVE_NS = Duration.ofSeconds(TIMEOUT).toNanos(); // nanos
+        final Socket s;
+        final int id;
+
+        // time at which the connection was accepted (nanos)
+        final long acceptTime;
+
+        // number of requests that had the expected URI
+        volatile int headerFound;
+        // time at which the worker thread exited
+        volatile long stopTime;
+        // Time at which the first call to is.read() for the last request
+        // returned. This includes cases where -1 was returned.
+        volatile long startReading;
+        // Lat time at which a byte was read from the stream.
+        volatile long lastReading;
+        // number of times that the time between two consecutive received requests
+        // exceeded the KEEP_ALIVE timeout.
+        volatile int timeoutExceeded;
+        // Number of requests handled by this worker
+        volatile int requestHandled;
```

Comment:
not used

Reply:
yeah. it happens to be the same thing than headerFound, unless there's an exception while sending the reply.